dspace-server-webapp/src/main/java/org/dspace/app/rest/submit/step/DescribeStep.java

```diff
@@ -45,6 +46,10 @@ public class DescribeStep extends AbstractProcessingStep {
 
     private DCInputsReader inputReader;
 
+    private static final String DOCUMENT_TYPE_FIELD =
```

Comment:
Could we move this out of a static global field?  Otherwise, I believe this value will be cached until you next restart Tomcat (which means this config won't reload automatically).  Instead, I'd recommend just loading this config where you need it in the methods below -- there's only one usage, so it should be a quick fix.

Reply:
Done!